data.sql

```diff
@@ -1,5 +1,60 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES 
+  ('Charmander', '2020-02-08', 0, 'no', -11.0),
+  ('Plantmon', '2021-11-15', 2, 'yes', -5.7),
+  ('Squirtle', '1993-04-02', 3, 'no', -12.13),
+  ('Angemon', '2005-06-12', 1, 'yes', -45.0),
+  ('Boarmon', '2005-06-07', 7, 'yes', 20.4),
+  ('Blossom', '1998-10-13', 3, 'yes', 17.0),
```

Comment:
- [ ] Great job so far please kindly add a  specializations table to complete the data according to the project requirment 


`-- Insert into specializations
INSERT INTO specializations (species_id, vet_id) 
VALUES(1,1), (2,3), (1,3), (2,4);`